[Term Entry] Java Calendar getDisplayName() (#3026)
Codecademy/docs: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md

---
Title: '.getDisplayName()'

> Description: 'Returns a String containing calendar field information based on input parameters'

Reviewer: ```suggestion
Description: 'Returns a string containing calendar field information based on the given input parameters.'
```

Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Calendar'
  - 'Methods'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The `.getDisplayName()` method of the `Calendar` class is used to return the string representation of a calendar field value in the given style and locale.

Reviewer: ```suggestion
The **`.getDisplayName()`** method of the `Calendar` class is used to return the string representation of a calendar field value in the given style and locale.
```

## Syntax

```pseudo

public String getDisplayName(int field, int style, Locale locale)

```

> Where `field` is of integer type and refers to the field of the calendar on which the operation is to be performed,  `style` is of integer type and refers to the style that is supposed to be applied to the string representation, and `locale` is of Locale object type and refers to the locale representing the string.

Reviewer: ```suggestion
Where:

 -  `field` an integer that refers to the field of the calendar on which the operation is to be performed.
 -  `style` an integer that refers to the style that is supposed to be applied to the string representation.
 -  `locale` is a Locale object type and refers to the locale representing the string.
```

## Example

The following example displays the day of the week.

```java

> 

Reviewer: remove empty line
